Comms - Data Warehouse launch (#209)
PostHog/meta: requests-for-comments/2024-06-21-warehouse-launch-plan.md

## [Marketing] Data warehouse launch plan

The data warehouse has been in an opt-in beta for a while, and has now been opened up to a public open beta. It'll be at least a few weeks before it launches (we need to give current users two weeks notice of pricing changes) BUT that means it's the perfect time to plan. 

### Ahead of launch

- [x] Publish a [Stripe data warehouse tutorial](https://github.com/PostHog/posthog.com/issues/8662) - Marketing
- [x] Publish a [Hubspot data warehouse tutorial](https://github.com/PostHog/posthog.com/issues/8664) - Marketing
- [x] Publish a [Zendesk data warehouse tutorial](https://github.com/PostHog/posthog.com/pull/8858) - Marketing
- [ ] Create dashboard templates for 2x common usecases - @ivanagas
- [x] Update docs as needed - Marketing
- [ ] Add Data Warehouse to in-app 'Explore the docs' menu - https://github.com/PostHog/posthog/pull/22715
- [x] Key art - @lottiecoxon (we have some already, open to new though)

### Launch day activity

> - [ ] Publish a launch blog piece about why data warehouse is important - @joethreepwood 

Author: ```suggestion
- [ ] https://github.com/PostHog/posthog.com/pull/8887 - @joethreepwood 
```

> - [ ] Update pricing page with data warehouse pricing - Growth

Author: ```suggestion
- [x] Update pricing page with data warehouse pricing - Growth
```

- [x] Update in-app onboarding with data warehouse option - Growth
- [ ] Share blogpost to HackerNews - Marketing
- [ ] https://github.com/PostHog/posthog.com/pull/8875 - @joethreepwood 
- [ ] https://github.com/PostHog/meta/issues/213 - @joethreepwood

> - [ ] Social push, changelog, Pylon announcement - @joethreepwood

Author: ```suggestion
```

### Follow-on activity

> - [ ] Case study: https://github.com/PostHog/posthog.com/issues/8684 - @joethreepwood

Author: ```suggestion
- [x] Case study: https://github.com/PostHog/posthog.com/issues/8684 - @joethreepwood
```

## [Marketing] Data warehouse launch plan

The data warehouse has been in an opt-in beta for a while, and has now been opened up to a public open beta. It'll be at least a few weeks before it launches (we need to give current users two weeks notice of pricing changes) BUT that means it's the perfect time to plan. 

### Ahead of launch

- [x] Publish a [Stripe data warehouse tutorial](https://github.com/PostHog/posthog.com/issues/8662) - Marketing
- [x] Publish a [Hubspot data warehouse tutorial](https://github.com/PostHog/posthog.com/issues/8664) - Marketing

> - [ ] Publish a [Zendesk data warehouse tutorial](https://github.com/PostHog/posthog.com/issues/8663) - Marketing

Author: ```suggestion
- [x] Publish a [Zendesk data warehouse tutorial](https://github.com/PostHog/posthog.com/pull/8858) - Marketing
```

- [ ] Create dashboard templates for 2x common usecases - @ivanagas
- [x] Update docs as needed - Marketing
- [ ] Add Data Warehouse to in-app 'Explore the docs' menu - https://github.com/PostHog/posthog/pull/22715
- [x] Key art - @lottiecoxon (we have some already, open to new though)

### Launch day activity

- [ ] Publish a launch blog piece about why data warehouse is important - @joethreepwood 

> - [ ] Update pricing page with data warehouse pricing
> - [ ] Update in-app onboarding with data warehouse option

Author: ```suggestion
- [ ] Update pricing page with data warehouse pricing - Growth
- [ ] Update in-app onboarding with data warehouse option - Growth
```

- [ ] Share blogpost to HackerNews - Marketing

> - [ ] https://github.com/PostHog/meta/issues/213 - @joethreepwood

Author: ```suggestion
- [ ] https://github.com/PostHog/posthog.com/pull/8875 - @joethreepwood 
- [ ] https://github.com/PostHog/meta/issues/213 - @joethreepwood
```

- [ ] https://github.com/PostHog/meta/issues/213 - @joethreepwood
- [ ] Social push, changelog, Pylon announcement - @joethreepwood

### Follow-on activity

- [ ] Case study: https://github.com/PostHog/posthog.com/issues/8684 - @joethreepwood

> - [ ] Update onboarding email flow with data warehouse options - @joethreepwood

Author: ```suggestion
```
Won't have this at launch

> - [ ] Invite Theo/Danny to try the warehouse and give what they need for a video on it - @ivanagas

Author: @ivanagas We're currently targeting a July 8th launch. Was there any chance of this?

Reviewer: I didn't follow up on this because I thought we wanted the launch to be quiet, this is going to show it to a lot of people, so I'm hesitant.

Author: Yep, all good - lets remove then.

Author: ```suggestion
```

## [Marketing] Data warehouse launch plan

The data warehouse has been in an opt-in beta for a while, and has now been opened up to a public open beta. It'll be at least a few weeks before it launches (we need to give current users two weeks notice of pricing changes) BUT that means it's the perfect time to plan. 

### Ahead of launch

> - [ ] Publish a Stripe data warehouse tutorial - Marketing
> - [ ] Publish a Hubspot data warehouse tutorial - Marketing
> - [ ] Publish a Postgres data warehouse tutorial - Marketing

Reviewer: Because Postgres is so flexible, it makes it really tricky to write a tutorial about anything specific
```suggestion
- [ ] Publish a [Stripe data warehouse tutorial](https://github.com/PostHog/posthog.com/issues/8662) - Marketing
- [ ] Publish a [Hubspot data warehouse tutorial](https://github.com/PostHog/posthog.com/issues/8664) - Marketing
- [ ] Publish a [Zendesk data warehouse tutorial](https://github.com/PostHog/posthog.com/issues/8663) - Marketing
```

- [ ] Update docs as needed - Marketing

> - [ ] Add Data Warehouse to in-app 'Explore the docs' menu - @joethreepwood

Author: Have asked Website how to do this, will take care of.

Author: ```suggestion
- [ ] Add Data Warehouse to in-app 'Explore the docs' menu - https://github.com/PostHog/posthog/pull/22715
```

> - [ ] Create dashboard templates for 2x common usecases - @ivanagas

Author: @ivanagas I see we've got one ready, but not sure if it's internal or not. Just wanted to check if we should delete this, or not?

Reviewer: I should have the first one ready today, but just internal at the moment.

- [x] Key art - @lottiecoxon (we have some already, open to new though)

### Launch day activity

> - [ ] Publish a launch blog piece about why data warehouse is important - 

Author: ```suggestion
- [ ] Publish a launch blog piece about why data warehouse is important - @jamesefhawkins 
```

@jamesefhawkins - Did you want to write something for this, or easier if I take care of it?

> - [ ] Publish a launch blog piece about why data warehouse is important - 

Reviewer 2: This could go in hand with the updated mission around equipping every developer - data warehouse is pretty relevant

> - [ ] Update pricing page with data warehouse pricing
> - [ ] Update in-app onboarding with data warehouse option

Author: ```suggestion
- [ ] Update pricing page with data warehouse pricing
- [ ] Update in-app onboarding with data warehouse option
```

I think this would sit with you @corywatilo / @raquelmsmith ? More of an FYI and feedback invite

> - [ ] Share blogpost to HackerNews - Marketing

Reviewer 2: Do we want to do this as a launch HN type thing @jamesefhawkins or are we just going to get destroyed here because it's super early.

> - [ ] Publish a launch blog piece about why data warehouse is important - 

Author: ```suggestion
- [ ] Publish a launch blog piece about why data warehouse is important - @joethreepwood 
```

> - [ ] Update in-app onboarding with data warehouse option - Growth

Author: @raquelmsmith I assume we're doing this. Can you let me know when it's live/when the PR is ready? I'll factor it into an onboarding email flow too.

Reviewer 3: For now I'm just going to put a button in the initial /products page that points them to the link sources page.

Author: ```suggestion
- [x] Update in-app onboarding with data warehouse option - Growth
```

- [ ] Share blogpost to HackerNews - Marketing

> - [ ] Email announcement to all users - @joethreepwood

Author: ```suggestion
- [ ] https://github.com/PostHog/meta/issues/213 - @joethreepwood
```

- [ ] Social push, changelog, Pylon announcement - @joethreepwood

> - [ ] Bookface launch to the YC community - @jamesefhawkins 

Author: ```suggestion
```

- [ ] Source a data warehouse customer story - @joethreepwood
- [ ] Update onboarding email flow with data warehouse options - @joethreepwood

### Bigger ideas

**Dashboard Templates**

> Is it possible to create dashboard templates that integrate with our integrated sources? E.g. a dashboard for getting Stripe data. 

Author: Question for @EDsCODE

Reviewer: Since data warehouse insights work like normal insights, dashboard templates should work.

Author: Awesome. Think we can get these built ahead of launch? 

It would be awesome if we had tutorials which walked through how to create some dashboards but then also pointed to templates as an example.

Reviewer: Yes

> - [ ] Source a data warehouse customer story - @joethreepwood

Author: ```suggestion
- [ ] Case study: https://github.com/PostHog/posthog.com/issues/8684 - @joethreepwood
```

**Discount Pricing**
We could offer discounted pricing for the first year to all users who have connected a source during the beta. Easy way to get good will and retain these users - and, if announced early, encourage some early adopters. 

**Data warehouse as a newsletter theme**
I've learned not to suggest anything too specific here. But talking about data warehouses in the newsletter is a way to get attention. 

> **Invite Theo to try it**

Reviewer 2: Danny has a similar reach and is using it/talking about it already so maybe a better fit?

Author: Sweet. I'll move this to the 'After launch' section. We want it to be out of beta for this, but timing isn't the be-all and end-all.

> We could invite Theo to try the data warehouse and talk about his thoughts. He could have someone to interview, etc., if he wants. 

Reviewer: I'll ask right now.

> We could invite Theo to try the data warehouse and talk about his thoughts. He could have someone to interview, etc., if he wants. 

Reviewer: He says yes, but we need to figure out exactly what the video will look like.

**Sponsorships**

> There are some newsletter sponsorships we've looked at before, such as DataElixir, which may be a good match for this audience. We could draw up a list of podcasts and newsletters to reach to ahead of time. (Suggestions welcome)

Reviewer 4: I could look into this. Think this should go into follow-on activity.

Author: LMk if any assist needed here, as we're expecting to launch next week.

**Integrate with upcoming PR announcement**
Being coy on details in this public repo, but have raised with James directly for discussion. 

**ProductHunt Launch**

> Not really, don't worry Charles. 

Reviewer 2: GOOD

## [Marketing] Data warehouse launch plan

The data warehouse has been in an opt-in beta for a while, and has now been opened up to a public open beta. It'll be at least a few weeks before it launches (we need to give current users two weeks notice of pricing changes) BUT that means it's the perfect time to plan. 

### Ahead of launch

- [ ] Publish a Stripe data warehouse tutorial - Marketing
- [ ] Publish a Hubspot data warehouse tutorial - Marketing
- [ ] Publish a Postgres data warehouse tutorial - Marketing
- [ ] Update docs as needed - Marketing
- [ ] Add Data Warehouse to in-app 'Explore the docs' menu - @joethreepwood
- [x] Key art - @lottiecoxon (we have some already, open to new though)

> 

Reviewer 5: Some more ideas for tutorials here:

- Joining your [stripe/hubspot/zendesk/etc] data with PostHog for filtering within the product
- Using your postgres app data in PostHog

Author: Great. I'm going to defer to @andyvan-ph @ivanagas and @Lior539 on how we tackle these tutorials and when. Hopefully we can get a few of these out ASAP.

Reviewer 4: From the sprint chat, sounds like we should just focus on the tutorial + template stuff for now and not worry too much about promotional activity just yet.

@Lior539 @ivanagas Shall we nail down a list of tutorials we want to focus on first, and divide them up?

Would be good to have your input here @EDsCODE, but is it more useful to have:

1. Very straightforward guides like "How to integrate and query your Stripe data"
2. Or more problem orientated ones like "How to analyze Google Ad ROI in PostHog"

Initially, I was thinking the latter, but maybe it's better to start with very simple setup guides, then follow-up for more specific "use case" ones?

In any case, I think we could end doing a lot of work here as users will need a lot more hand holding to learn how to use warehouse / why it could be useful for them.

Reviewer: Lets do:
- How to set up Stripe reports
- How to set up Zendesk reports
- How to set up Hubspot reports

Each of these have solid search and relatively light competition. We can use them to:
1. Build a related dashboards which then we can convert to a template
2. Do an example in each of joining and combining source and product data

Reviewer: I'll write issues for all these today.

Reviewer: - Stripe: https://github.com/PostHog/posthog.com/issues/8662
- Zendesk: https://github.com/PostHog/posthog.com/issues/8663
- Hubspot: https://github.com/PostHog/posthog.com/issues/8664

Reviewer 4: I think we should do a Google Ad ROI at some point as well. Eric mentioned it as something people are doing already. Also, bonus point, we can set it up for ourselves. 😅

Reviewer: We should definitely prioritize tutorials that are useful in multiple ways. The ones I mention are useful as tutorials, SEO pieces, and help build dashboards. Google Ad ROI will be similar :)

### Launch day activity

- [ ] Publish a launch blog piece about why data warehouse is important - 
- [ ] Update pricing page with data warehouse pricing
- [ ] Update in-app onboarding with data warehouse option
- [ ] Share blogpost to HackerNews - Marketing
- [ ] Email announcement to all users - @joethreepwood
- [ ] Social push, changelog, Pylon announcement - @joethreepwood
- [ ] Bookface launch to the YC community - @jamesefhawkins 

### Follow-on activity

- [ ] Source a data warehouse customer story - @joethreepwood
- [ ] Update onboarding email flow with data warehouse options - @joethreepwood

### Bigger ideas

**Dashboard Templates**
Is it possible to create dashboard templates that integrate with our integrated sources? E.g. a dashboard for getting Stripe data. 

**Discount Pricing**
We could offer discounted pricing for the first year to all users who have connected a source during the beta. Easy way to get good will and retain these users - and, if announced early, encourage some early adopters. 

> **Data warehouse as a newsletter theme**

Author: @jtemperton

Reviewer 4: I think something along the lines of "How / Why we decided to build a data warehouse". It would need to get specific, but people enjoy hearing how we decide what to build.

Reviewer 4: Ian is working on something for the website atm: https://github.com/PostHog/posthog.com/issues/8646

Will see how this comes out. Based on the outline, I think we could base a newsletter on elements on Ian's planned article, probably without the more technical bit around how it works.
